sys/net/gnrc/conn/gnrc_conn.c

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
```

Comment:
I think none (or very few) C files written by @authmillenon have an author. Since doxygen is optional for C files, I think this is fine.